Library/Formula/collada-dom.rb

```diff
@@ -3,33 +3,45 @@
 class ColladaDom < Formula
   desc "C++ library for loading and saving COLLADA data"
   homepage 'http://www.collada.org/mediawiki/index.php/Portal:COLLADA_DOM'
```

Comment:
Suggesting to follow the redirect and move to 'https://':
https://www.khronos.org/collada/wiki/Portal:COLLADA_DOM